src/taretto/application_skel/{{cookiecutter.application_name}}/{{cookiecutter.application_name}}/base/application/__init__.py

```diff
@@ -0,0 +1,43 @@
+import importscan
+
+from pkg_resources import iter_entry_points
+from taretto.modeling import EntityCollections
```

Comment:
This import is breaking with the use of `apipkg.initpkg`, as this modifies the namespace and results in only `taretto.ui` and `taretto.navigate` resolving when taretto is installed as a dependency.

I played with adding `modeling` to the exported namespace, but class imports from the module failed.

```
❯ git diff
diff --git a/src/taretto/__init__.py b/src/taretto/__init__.py
index b39d42a..4aa38c9 100644
--- a/src/taretto/__init__.py
+++ b/src/taretto/__init__.py
@@ -2,6 +2,7 @@ from apipkg import initpkg as __initpkg
 
 # fmt: off
 __initpkg(__name__, {
+    "modeling": "taretto.modeling",
     "navigate": "navmazing",
     "ui": {
         "core": "widgetastic.widget",  # todo: html widget library here, limit to base
```

```
❯ pip freeze |grep taretto
-e git+ssh://git@github.com/mshriver/taretto.git@3c102517377b4f994fbb3fe421de5ea3cbf6dd7f#egg=taretto
❯ ipython
Python 3.8.12 (default, Aug 30 2021, 00:00:00) 
Type 'copyright', 'credits' or 'license' for more information
IPython 8.0.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: from taretto import modeling

In [2]: from taretto.modeling import EntityCollections
Fatal Python error: Cannot recover from stack overflow.
Python runtime state: initialized

Thread 0x00007fe5cba00640 (most recent call first):
  File "/usr/lib64/python3.8/threading.py", line 302 in wait
  File "/usr/lib64/python3.8/threading.py", line 558 in wait
  File "/home/mshriver/repos/satqe/robottelo/.robottelo/lib64/python3.8/site-packages/IPython/core/history.py", line 829 in run
  File "/home/mshriver/repos/satqe/robottelo/.robottelo/lib64/python3.8/site-packages/IPython/core/history.py", line 60 in only_when_enabled
  File "/home/mshriver/repos/satqe/robottelo/.robottelo/lib64/python3.8/site-packages/decorator.py", line 232 in fun
  File "/usr/lib64/python3.8/threading.py", line 932 in _bootstrap_inner
  File "/usr/lib64/python3.8/threading.py", line 890 in _bootstrap

Current thread 0x00007fe5dad43740 (most recent call first):
  File "/home/mshriver/repos/satqe/robottelo/.robottelo/lib64/python3.8/site-packages/apipkg/__init__.py", line 148 in importobj
  File "/home/mshriver/repos/satqe/robottelo/.robottelo/lib64/python3.8/site-packages/apipkg/__init__.py", line 290 in getmod
  File "/home/mshriver/repos/satqe/robottelo/.robottelo/lib64/python3.8/site-packages/apipkg/__init__.py", line 305 in __getattribute__
  <loop on above 3 lines>
  ...
[1]    324960 IOT instruction (core dumped)  ipython


```